src/test/java/com/unstoppabledomains/resolution/ENSTest.java

```diff
@@ -18,15 +18,15 @@ public void initEach() {
   }
 
   @Test
-  public void simpleTest() throws NamingServiceException {
+  public void simpleTest() throws NamingServiceException, Exception {
     String owner = resolution.owner("monkybrain.eth");
     assertEquals("0x842f373409191cff2988a6f19ab9f605308ee462", owner);
 
     TestUtils.checkError(() -> resolution.email("monkybrain.eth"), NSExceptionCode.RecordNotFound);
   }
 
   @Test
-  public void addressTest() throws NamingServiceException {
+  public void addressTest() throws NamingServiceException, Exception {
```

Comment:
`NamingServiceException` can be removed, since `Exception` is a broader type.

Reply:
Ops, I forgot to remove it. This was needed for TestUtils.checkError method, and I have moved them into a separate testError